### PR TITLE
[DC-3210] [DC-3212] Update RDR QC notebooks for `death`

### PR DESCRIPTION
**Pull request by hiro-mishima**

### CODE REVIEW

**Review by brendagutman — CHANGES_REQUESTED** [review]
I had a few questions. Otherwise looks really good. Thanks for consolidating the queries, that should help with the organization of this notebook.

File: data_steward/analytics/cdr_ops/raw_rdr_export_qc.py [hunk 1]
```diff
@@ -791,3 +791,69 @@
...
+
+query = JINJA_ENV.from_string("""
+SELECT
+    person_id
+FROM `{{project_id}}.{{dataset}}.death`
+WHERE death_type_concept_id != 32809
+OR cause_concept_id IS NOT NULL
+OR cause_source_value IS NOT NULL
+OR cause_source_concept_id IS NOT NULL
+-- OR src_id != 'healthpro' --
```
[the remark this answers is not in the document]
Comment (hiro-mishima) [reply data_steward/analytics/cdr_ops/raw_rdr_export_qc.py]: Thanks for pointing it out! It was meant to be uncommented. For the pass/fail statement, the `render_message()` on line 855 will take care of it.

And clean_rdr_export_qc.py does not need the condition. The death table will not have a `src_id` column after the clean RDR process. 
The definition of `death` when we receive it from RDR -> [here](https://github.com/all-of-us/curation/blob/develop/data_steward/resource_files/schemas/rdr_src_id/tables/rdr_death.json)
The definition of `death` at the time we run clean_rdr_export_qc.py -> [here](https://github.com/all-of-us/curation/blob/develop/data_steward/resource_files/schemas/cdm/clinical/death.json)

[the remark this answers is not in the document]
Comment (brendagutman) [reply data_steward/analytics/cdr_ops/raw_rdr_export_qc.py]: Thank you for the very thorough explanation!

**Review by brendagutman — APPROVED** [review]
Looks good Hiro!

**Pull request merged**